packages/editor/src/store/test/selectors.js

```diff
@@ -1362,25 +1411,17 @@ describe( 'selectors', () => {
 					title: 'sassel',
 				},
 				saving: {},
+				getCurrentUser() {},
```

Comment:
It looks like those functions are added here only to satisfy custom implementation of `selectors`. I think we should explore some alternatives to make those tests easier to read.

Reply:
>https://github.com/WordPress/gutenberg/pull/16932#discussion_r313019176

I agree it makes the tests harder to reason about at first glance. But, I think it's a fine trade-off against rewriting all of them in this PR without really gaining extra coverage or a better harness.